couchdb/source/config/500-webhook-configuration.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: webhook.couchdb.messaging.knative.dev
+  name: defaulting.webhook.couchdb.messaging.knative.dev
```

Comment:
@mattmoor @vaikas  are changes like this something that we should add in the release notes ?